Models/Project.cs

```diff
@@ -75,6 +75,9 @@ public Project()
 
         public bool InstitutePrivate { get; set; }
 
+        public List<ProjectCategory> Categories { get; set; } = new List<ProjectCategory>();
```

Comment:
This new should be deleted I think

Reply:
Why?

Reply:
Why not? I don't see it at one of the other properties?